pages/[id]/FeatureFlags.js

```diff
@@ -0,0 +1,197 @@
+import styled from 'styled-components';
+import { useEffect, useState } from 'react';
+import Page from '../../components/page';
+import Card, {
+  CardHeader,
+  CardButtonContainer,
+  CardTitle,
+  CardContent,
+} from '../../components/card';
+import Button from '../../components/button';
+import { COLOR, EDIT } from '../../constants';
+import {
+  getTimestamp,
+  subscribeToFlags,
+  formatDate,
+  updateFlags,
+  getHackathonPaths,
+  getHackathons,
+} from '../../utility/firebase';
+import { useAuth } from '../../utility/auth';
+
+const CancelText = styled.p`
+  border-bottom: 2px solid ${COLOR.BLACK};
+  margin: 0px;
+`;
+
+const CancelButton = styled.button`
+  font-size: 16px;
+  cursor: pointer;
+  border-bottom: 2px solid ${COLOR.BLACK};
+  margin-left: 675px;
+  margin-right: 40px;
+  border: none;
+  outline: none;
+  background-color: ${COLOR.BACKGROUND};
+`;
+
+const FeatureFlagsContainer = styled.div`
+  display: flex;
+  width: 40%;
+  border: 2px solid lightgray;
+  margin-top: 10px;
+  margin-bottom: 10px;
+  justify-content: space-between;
+  background-color: #ededed;
+`;
+
+const FeatureFlagToggle = styled.input`
+  width: 18px;
+  height: 18px;
+  margin-top: 10px;
+`;
+
+const FeatureFlagName = styled.p`
+  font-weight: bold;
+  font-size: 12px;
```

Comment:
```suggestion
  font-weight: ${(props) => (props.isLabel ? '600' : '400')};
  font-size: 16px;
  color: ${COLOR.BODY_TEXT};
```
On the FAQ and spocos pages, I remember the font size was 16px and the colour was slightly lighter so this might look more consistent
also the font weight thing could make it easier for the user to differentiate label from indicator text ![Screen Shot 2020-09-17 at 11 08 03 PM](https://user-images.githubusercontent.com/38872354/93562721-73d5ce80-f93b-11ea-9bd0-a66e396c9b70.jpg)